app/models/micropost.rb

```diff
@@ -16,6 +16,9 @@ class Micropost < ApplicationRecord
   belongs_to :user
 
   scope :recent_posts, -> {order(created_at: :desc)}
+  scope :relate_post, lambda {|user_ids|
+    where(user_id: user_ids).order(created_at: :desc)
+  }
```

Comment:
```suggestion
  scope :relate_post, lambda do |user_ids|
    where(user_id: user_ids).order(created_at: :desc)
  end
```
block nhiều dòng thì dùng do .. end e nha